### ISSUE TITLE
Add Custom Links to Images in Basic Gallery - Image Gallery Dropdown

### ISSUE THREAD

**Issue by KevinLittle** [elementor/elementor#9692]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
In the Basic Gallery (image gallery) - There is a link drop-down. You give options for a link to Media File, or a link to the Attachment Page itself. But there is no option for a custom link, unless we want to custom code... Which is why we have Elementor in the first place, to avoid marketing teams having to make requests from developers.

**Describe the solution you'd like**
My company would really appreciate a way to add a custom link to this field in Basic Gallery > Image Gallery, so that a user can click on an image and be taken to a corresponding page.

**Describe alternatives you've considered**
The best we can do is attempt to modify code to allow custom links in the dropdown.

**Additional context**
This seems more like a general oversight than an intentional mistake - being able to add a custom link destination on an image to a custom link is very basic. I've attached an image to show the area I'm talking about - perhaps this is a feature in an upgraded plan?

![Example-Elementor-Gallery](https://user-images.githubusercontent.com/5853300/69655150-f7889c80-1043-11ea-86c7-3f30b7536669.png)

**Comment by philipellis** [comment]
YES! I need to be able to link EVERY IMAGE in my gallery with a unique link. Thinking that 1 link will work for all the images seems like a major oversight.
